### DISCUSSION TITLE
Default max username length seems excessive

### DISCUSSION THREAD

**Discussion by powerjungle** [Syncplay/syncplay#596] [Ideas]
Is there a reason for the default username length to be 150 characters? When a video is played and somebody makes an username, longer then say 10 or 15 characters it takes up a big portion of the player interface. I have a feeling a value of 10 or 15 is more logical.

I'm refering to this line:

https://github.com/Syncplay/syncplay/blob/9be87bc9bfa31903c65fde67804fc5f0883261ba/syncplay/constants.py#L99

**Comment by Et0h** [comment]
Thanks for bringing this to my attention and pointing to the relevant line of code.

You've right. It does seem excessive.

I think the max username length should be reduced back to its historic setting of 16 characters and I'm not sure why it was ever increased from that level.

I am not 100% sure but my best guess is that it was accidentally changed when trying to manually merge an upstream change which increased chat length to 150 characters but somehow increased not just chat length but also username length.

Prior to https://github.com/Syncplay/syncplay/commit/6f172d9803d44d9989354ddad536d9536e376f61 the values were 

MAX_CHAT_MESSAGE_LENGTH = 125 # Number of displayed characters
MAX_USERNAME_LENGTH = 20 # Number of displayed characters

Before it was 20 it was 16. I can't remember why it was increased to 20 but I can't remember how the character count handles Unicode characters so maybe that was the reason. The change at https://github.com/Syncplay/syncplay/commit/f5c6b8fcf998835f6976f06d2b0ebd0a2fa768cc doesn't mention increasing the username limit from 16 to 20 so I wonder if it was just something I did to test if it properly handled a higher value but it ended up getting pushed by mistake.

I think the Unicode issue was fixed by your change at https://github.com/Syncplay/syncplay/pull/506 so  even if that was the reason I think we should now revert to 16 based on the consensus view from the poll at https://github.com/Syncplay/syncplay/issues/160

**Comment by powerjungle** [comment]
Yep, trying to make an username with ❤️ emojis shows up only 5 characters when the limit is set to 10. ASCII characters get cut to the correct limit of 10. Trying it with cyrilic characters cuts to the correct limit of 10 again. I have a feeling setting it around 15 would be good enough.

**Comment by Et0h** [comment]
Okay, I've now reverted it to 16 characters as per #160. Even if people have less than 16 displayed characters due to using a Unicode language it should still be enough for them to have a unique and recognisable name.
